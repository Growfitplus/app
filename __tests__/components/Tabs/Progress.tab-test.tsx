import ProgressTab from '@/components/Tabs/Progress.tab';
import { render } from '@testing-library/react-native';

describe('Progress Tab', () => {
  it('should render without crashing', () => {
    const { toJSON } = render(<ProgressTab focused />);

    expect(toJSON()).toMatchSnapshot();
  });

  it('should not add bg when not focused', () => {
    const { toJSON } = render(<ProgressTab focused={false} />);

    expect(toJSON()).toMatchSnapshot();
  });
});
