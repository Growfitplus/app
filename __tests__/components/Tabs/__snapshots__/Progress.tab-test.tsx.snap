// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Progress Tab should not add bg when not focused 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "none",
      "borderRadius": 14,
      "height": 40,
      "justifyContent": "center",
      "width": 40,
    }
  }
>
  <Text />
</View>
`;

exports[`Progress Tab should render without crashing 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#00FFC2",
      "borderRadius": 14,
      "height": 40,
      "justifyContent": "center",
      "width": 40,
    }
  }
>
  <Text />
</View>
`;
